WTWSMS/common/traits/VIET_Immersion_Other.txt

```diff
@@ -330,6 +316,13 @@ on_retreat = {
 
 #Chinese traits
 son_of_heaven = {
+	potential = {
+		OR = {
+			culture_group = sinic
+			religion = confucian
+		}
+		higher_real_tier_than = KING
+	}
```

Comment:
Yes, but this trait can not be acquired through other means than the existant flavour.